mkdocs.yml

```diff
@@ -0,0 +1,46 @@
+site_name: pynetbox
+site_description: Python API client library for NetBox
+theme:
+  name: material
+  features:
+    - navigation.tabs
+    - navigation.sections
+    - navigation.expand
+    - navigation.top
+
+plugins:
+  - search
+  - mkdocstrings:
+      default_handler: python
+      handlers:
+        python:
+          options:
+            show_source: true
+            show_root_heading: true
+
+nav:
+  - Home: index.md
+  - API Reference:
+    - Endpoint: endpoint.md
+    - Response: response.md
+    - Request: request.md
+    - IPAM: IPAM.md
```

Comment:
What about other apps?

Reply:
Add as a new issue, I want to keep this with just converting the documentation to replicate current documentation.